requirements.txt

```diff
@@ -1,6 +1,8 @@
+coverage
+dateutils
+pymongo>=3.5.1
+pytest
+requests>=2.13.0
+six>=1.10.0
```

Comment:
I think we can get rid of this file and just have our requirements in setup.py. No point maintaining versions in two places

Reply:
I think this file is used in the Dockerfile? I haven't looked into what the docker does, but I kept it in case it was important.